components/accordion/index.css

```diff
@@ -42,8 +42,11 @@ governing permissions and limitations under the License.
   position: absolute;
   inset-inline-start: var(--spectrum-accordion-item-padding-x);
   inset-block-start: calc(
-    var(--spectrum-accordion-item-height-actual) / 2 -
-      var(--spectrum-accordion-icon-height) / 2
+    50% -
+    var(
+      --spectrum-accordion-icon-height,
+      var(--spectrum-global-dimension-size-125)
```

Comment:
@lazd do you think we need this fallback?

Reply:
That fallback should be added automatically the the postcss-custom-properties-mapping plugin, so no.

Reply:
Updated! Cool to see that's managed automatically, I'll have to look deeper into the magic there. 🧙